specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/examples/package/createOrUpdatePackage.json

```diff
@@ -23,9 +22,9 @@
     "201": {
       "headers": {},
       "body": {
-        "id": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Automation/automationAccounts/myAutomationAccount33/modules/OmsCompositeResources",
+        "id": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Automation/automationAccounts/myAutomationAccount33/runtimeEnvironments/runtimeEnvironmentName/packages/OmsCompositeResources",
         "name": "OmsCompositeResources",
-        "type": "Microsoft.Automation/AutomationAccounts/Modules",
+        "type": "Microsoft.Automation/AutomationAccounts/RuntimeEnvironments/Packages",
```

Comment:
>RuntimeEnvironments [](http://example.com/codeflow?start=57&length=19)

Your service should not be uppercasing the resource type. Resource type names use camelCasing.

Reply:
I have updated the casing for resource type names to camelcasing.